__tests__/app.test.js

```diff
@@ -0,0 +1,50 @@
+const db = require('../db/connection')
+const data = require('../db/data/test-data/index')
+ const seed = require('../db/seeds/seed')
+ const { app } = require('../app')
+ const request = require('supertest')
+const { string } = require('pg-format')
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+
+describe('CORE: GET /api/topics', () => {
+    test('returns 200 status code', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+    });
+    test('returns an array', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response)=>{
+            const topics = response.body;
+            expect(Array.isArray(topics)).toBe(true);
+        })
+    });
+    test('returns an array of objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response)=>{
+            const topics = response.body;
```

Comment:
As we are using a forEach here - if the array is empty then this test will pass by default as none of the expect statements will be called. A test with no expect statements will pass by default. We should first check that there is something in the array (check its length)